run_brer/plugin_configs.py

```diff
@@ -124,6 +124,8 @@ def build_plugin(self):
         """
         if self.get_missing_keys():
             raise KeyError('Must define {}'.format(self.get_missing_keys()))
+        if self.get('alpha') == 0:
+            raise RuntimeError('Alpha value is constrained to 0.')
```

Comment:
These RuntimeErrors should probably be `ValueError`. In any case, they need to be consistent with what we try to catch in the test.

Reply:
```suggestion
            raise ValueError('Read a non-sensical alpha value: 0.0')
```